bin/dev

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/ruby --disable-gems
+#!/usr/bin/env -S ruby --disable-gems
```

Comment:
hmm... `-S` isn't available in minimal linux installs but this seems like a good trick, more portable than insisting on `/usr/env/ruby` 🤷